src/Common/Commands.Common.Storage/Adapters/ARM.Storage.4/ARMStorageProvider.cs

```diff
@@ -12,9 +12,9 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Management.Storage;
```

Comment:
Common changes are not needed here. Should be done in common repo.

Reply:
done